.github/ISSUE_TEMPLATE/release.md

```diff
@@ -142,15 +142,4 @@ date to include them.
 
 ### Benchmarks
 
-Until #631 is done, running the benchmarks is a manual process:
-1. Connect to the bechmarking machine
-2. Make sure no one else is using the machine with `htop check`
-3. Pull in the branch of Cumulus that has the version of Statemine you want to release
-4. Recompile `cargo build --profile production --locked --features runtime-benchmarks`
-5. From the root directory run `nohup ./scripts/benchmarks.sh &` (it will take quite a few hours)
-6. Checkout in your local machine to the branch of cumulus that has the version of Statemine you want to release
-7. `scp` from the host to your local machine the weights for Statemine, Westmint and Statemint you'll find in:
-   - `/parachains/runtimes/assets/statemine/src/weights`
-   - `/parachains/runtimes/assets/statemint/src/weights`
-   - `/parachains/runtimes/assets/westmint/src/weights`
-8. Commit the changes in your local and create a PR
+The benchmarks are trigger from PRs against the `release-parachains-v*` branches.
```

Comment:
```suggestion
The benchmarks are triggered by PRs against the `release-parachains-v*` branches.
```

Reply:
why is this section in the issue template anyway

Reply:
This is a NOOP (almost) to (re)trigger the benchmarks. You will find what you have in mind in [this PR](https://github.com/paritytech/cumulus/pull/1297)